### PR TITLE
Add responsible disclosure policy

### PR DESCRIPTION
**Pull request by jkmassel**
- [x] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,6 +84,10 @@ that can't be shared publicly. More documentation and guides can be found on the
 
 Read our [Contributing Guide](CONTRIBUTING.md) to learn about reporting issues, contributing code, and more ways to contribute.
 
+## Security
+
+  Need to report a security vulnerability? Go to https://automattic.com/security/ or directly to our security bug bounty site https://hackerone.com/automattic.
```
Comment (designsimply) [inline README.md]:
```suggestion
If you happen to find a security vulnerability, we would appreciate letting us know at https://hackerone.com/automattic and allow us to respond before disclosing the issue publicly.
```

**Comment by jkmassel** [comment]
> I noticed there are separate HackerOne pages for Automattic and WordPress. Should `WordPress-Android` point to https://hackerone.com/wordpress? Note: if it's a preference and either will work then I could see choosing https://hackerone.com/automattic as it likely would have the most eyes on mobile development and get seen faster. I don't have a preference but I know some in the community may and it's worth a quick check.

Good question! From the WP HackerOne page:

>Please note that WordPress.com is a separate entity from the main WordPress open source project. Please report vulnerabilities for WordPress.com or the WordPress mobile apps through Automattic's HackerOne page.

**Comment by designsimply** [comment]
Haha, I don't think the last one was actually a repeat 😂 it was just a comment on the suggested change…

**Comment by designsimply** [comment]
> Good question! From the WP HackerOne page:
> 
> > Please report vulnerabilities for WordPress.com or the WordPress mobile apps through Automattic's HackerOne page.

Aha. Excellent. I missed that (but I like knowing it and I like that you caught it). 😊

File: README.md (hunk 1 above)
Comment (jkmassel) [reply README.md]: Can we change
> allow us

to 

> allowing us 

? 

😅

File: README.md [hunk 2]
```diff
@@ -84,6 +84,10 @@ that can't be shared publicly. More documentation and guides can be found on the
 
 Read our [Contributing Guide](CONTRIBUTING.md) to learn about reporting issues, contributing code, and more ways to contribute.
 
+## Security
+
+If you happen to find a security vulnerability, we would appreciate you letting us know at https://hackerone.com/automattic and allow us to respond before disclosing the issue publicly.
```
Comment (jkmassel) [inline README.md]: Can we change "allow us" to "allowing us"? 

Then good to :shipit:!

Comment (designsimply) [reply README.md]: 👍

**Pull request merged**